taca/deliver/cli.py

```diff
@@ -0,0 +1,100 @@
+""" CLI for the deliver subcommand
+"""
+import click
+from ngi_pipeline.utils.communication import mail_analysis
+from taca.deliver import deliver as _deliver
+
+@click.group()
+@click.pass_context
+@click.option('--deliverypath', type=click.STRING,
```

Comment:
this is just aesthetics and my opinion but, don't you think that it would be better to have `--delibery-path` and `--staging-path`, for example?